apps-rendering/src/item.ts

```diff
@@ -323,24 +327,33 @@ const isPicture = hasTag('type/picture');
 
 const fromCapiLiveBlog =
 	(context: Context) =>
-	(request: RenderingRequest): LiveBlog | DeadBlog => {
+	(
+		request: RenderingRequest,
+		blockId: Option<string>,
+	): LiveBlog | DeadBlog => {
 		const { content } = request;
 		const body = content.blocks?.body ?? [];
+		const pageSize = content.tags.map((c) => c.id).includes('sport/sport')
```

Comment:
We've taken the page size definition from frontend: https://github.com/guardian/frontend/blob/main/article/app/model/LiveBlogHelpers.scala#L45